front/src/modules/ui/utilities/recoil-scope/hooks/useRecoilScopedFamilyState.ts

```diff
@@ -4,18 +4,20 @@ import { RecoilState, useRecoilState } from 'recoil';
 import { RecoilScopeContext } from '../states/RecoilScopeContext';
 
 export function useRecoilScopedFamilyState<StateType>(
-  recoilState: (param: string) => RecoilState<StateType>,
-  stateKey: string,
+  recoilState: (familyUniqueId: string) => RecoilState<StateType>,
+  uniqueIdInRecoilScope: string,
   SpecificContext?: Context<string | null>,
 ) {
   const recoilScopeId = useContext(SpecificContext ?? RecoilScopeContext);
 
   if (!recoilScopeId)
     throw new Error(
       `Using a scoped atom without a RecoilScope : ${
-        recoilState(stateKey).key
+        recoilState('').key
       }, verify that you are using a RecoilScope with a specific context if you intended to do so.`,
     );
 
-  return useRecoilState<StateType>(recoilState(recoilScopeId + stateKey));
+  const familyUniqueId = recoilScopeId + uniqueIdInRecoilScope;
```

Comment:
Recoil's docs mention the family parameter can be an array of serializable values: https://recoiljs.org/docs/api-reference/utils/atomFamily/#parameter-type

So we could use a `[string, string]` array here.

Reply:
Really? I've been concatenating familyIds for a long time!
Let's create a discussion to refactor that